src/Actions/NotifyUsersWorkflowAction.php

```diff
@@ -86,9 +87,21 @@ public function fieldLabels($relations = true)
 
     public function execute(WorkflowInstance $workflow)
     {
+        if (!$this->EmailTemplate) {
+            Injector::inst()->get(LoggerInterface::class)->warning(
+                'No email template available for notify users workflow action '
+                . "'{$this->Title}' in workflow '{$this->WorkflowDef()?->Title}'."
+            );
+            return true;
+        }
+
         $members = $workflow->getAssignedMembers();
 
         if (!$members || !count($members ?? [])) {
+            Injector::inst()->get(LoggerInterface::class)->warning(
+                'No members to send emails to for notify users workflow action '
+                . "'{$this->Title}' in workflow '{$this->WorkflowDef()?->Title}'."
+            );
```

Comment:
This warning doesn't resolve any of the listed issues, but adds logging consistent with other logging I'm adding in this PR so that the notify users action can either be updated or removed as appropriate.